.gitignore

```
@@ -1,2 +1,5 @@
 # ignore nextflow logs
 .nextflow*
+
+# ignore work directory
+work*
```

Just to keep this more minimal (there aren't other work* files we want to ignore, are there?)
```suggestion
work/
```